### PR TITLE
App-project: Modify document title for About pages

### PR DESCRIPTION
**Pull request by goplayoutside3**
_Please request review from `@zooniverse/frontend` team or an individual member of that team._ 

Package:
app-project

Towards #2173.

Describe your changes:
Added a function to `HeadContainer` using `withRouter` to pass a `page` prop to the `Head` component. This prop can be used to modify the document title based on the current About page or Classify page for improved accessibility. 

@eatyourgreens and @srallen - Do you think the title should be even more specific than the addition of 'About' or 'Classify'? 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
I think Passing down the page title as a prop from each page under `app-project/pages` would be more flexible. For example, that would allow you to have individual titles for the team, FAQ and research pages.

Classify is absolutely fine for all the pages under `/classify` right now.  As part of Engaging Crowds, I am planning on adding workflow and subject set names to the page titles for individual workflows and sets.

I'd be wary of hard-coding English strings directly into components eg. here the page titles are 'Recherche' and 'Équipe'. I don't think the code needs to be localised right now, but it should be open to future localisation.
https://www.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/about/research?language=fr
https://www.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/about/team?language=fr

**Comment by goplayoutside3** [comment]
@eatyourgreens Totally understandable! I revised this PR so that `pageTitle` is passed under `app-project/pages` and uses specific names for the different `/about` pages. I'll leave the modification of document title under `/classify` to you 👍

File: packages/app-project/src/components/Head/HeadContainer.js [hunk 1]
```diff
@@ -52,6 +57,7 @@ class HeadContainer extends Component {
       <Head
         description={this.getProjectDescription()}
         ogImage={this.getProjectImage()}
+        pageTitle={this.getPageTitle()}
```
Comment (eatyourgreens) [inline packages/app-project/src/components/Head/HeadContainer.js:60]: Could this be `pageTitle={this.props.pageTitle}`? I'm not sure why it's calling a method that just returns a prop.

**Review by eatyourgreens — APPROVED** [review]
Looks good to me. All the About pages have individual titles now.
Screenshot of the team page for HMS NHS, showing that the title of the open tab is The Team | HMS NHS…

**Pull request merged**